forge/auditLog/formatters.js

```diff
@@ -40,8 +40,12 @@ const generateBody = ({ error, team, project, sourceProject, device, user, stack
     if (isObject(subscription)) {
         body.subscription = subscriptionObject(subscription)
     }
-    if (typeof license === 'string') {
-        body.license = license
+    if (license !== undefined) {
+        if (typeof license === 'string') {
+            body.license = license
+        } else {
+            body.license = license
```

Comment:
The license can be not a string?

Reply:
It can be a string, or full object, but awake me is now realising that if statement is entirely redundant